src/content/docs/network-performance-monitoring/troubleshooting/different-entity-same-ip.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Same IP for Multiple Entities
+tags:
+  - Integrations
+  - Network monitoring
+  - Troubleshooting
+metaDescription: You see multiple entities in the Explorer with the same IP, but different names.
+---
+
+## Problem [#problem]
+
+You see multiple entities in the **Explorer** menu with the same IP, but different names.
+
+This can happen when the device using that IP has been renamed, or the IP has moved to a different device.  Your `ktranslate snmp-base.yaml` file will have entries for every combination of `device_name__ip` it has ever discovered.  This is common when an IP is used for redundancy like with HSRP, switch stacks, or primary/secondary failovers.
+
+When a discovery job runs, each time it discovers an IP it tries to determine the `device_name` connected to the IP address.  If there is not already an entry in the devices list that matches that specific combination of `device_name__ip` then it will add a new entry with a hard coded `device_name`.  It does not automatically over write or remove previous entries that used that IP.  
+
+Another example of redundancy can occur when `ktranslate` performs its polling. `Ktranslate` only uses the IP when it runs, so if your devices list contains multiple entries with the same IP address, it will collect and send metrics to New Relic as if they were separate entities, but in reality it is just the same data that came from whatever device responded to requests to IP at the current polling interval.  The `device_name` is not collected or updated as part of the polling cycle.
```

Comment:
This is not another example related to redundancy, this is just explaining how ktranslate works all the time.  The only part where we need to bring up the redundancy i think is the list of protocols at the end of line 14.